lib/beautiful/xresources.lua

```diff
@@ -75,8 +75,8 @@ end
 -- @treturn number DPI value.
 function xresources.get_dpi(s)
     s = get_screen(s)
-    if dpi_per_screen[s] then
-        return dpi_per_screen[s]
+    if s then
+        return dpi_per_screen[s] or s.dpi
```

Comment:
And the next problem: People who have wrong dimensions in their EDID data and overrode this via `Xft.dpi` will now have `Xft.dpi` being ignored. Since `Xft.dpi` is some explicit configuration, I think it should have higher priority than auto-configuration.

Meaning that I would propose the following order of settings:
- Forced settings via Lua: `dpi_per_screen[s]`
- Forced settings via xrdb: `Xft.dpi`
- Automatically computed from outputs: `s.outputs`
- Automatically computed from the size of the root window (the code below)
- Good old `96`